view/src/components/produce.js

```diff
@@ -230,10 +220,112 @@ class produce extends Component {
 		});
 	}
 
+	// This function updates the string that the produce array will be searched for.
+	// This string value can contain any produce field (name, weight, interrnal shipment numbers) and will still return
+	// 	the appropriate filtered results.
+
 	handleSearch = event => {
-		const { value } = event.target;
-		this.setState({ value });
-	  };
+    const { value } = event.target;
+    this.setState({ value });
+	};
+
+	// Handles the rendering of filtered produce results into React cards
+	handleResultsRender = (results) => {
+		const { classes } = this.props;
+		return (
+			<div>
+			{results.map((produce) => (
+			<Grid item xs={12} key={produce.produceId}>
+				<Card className={classes.root} variant="outlined">
+				<CardContent>
+					<Typography variant="h5" component="h2">
+					{produce.name}
+					</Typography>
+					<Box
+					display="flex"
+					flexDirection="row"
+					flexWrap="wrap"
+					p={0}
+					m={0}
+					>
+					<Box p={3}>
+						<Typography
+						className={classes.pos}
+						color="textSecondary"
+						>
+						Shipping Temperatures in Reefer (°F):
+						</Typography>
+						<Typography variant="body2" component="p">
+						Maintenance Temperature:{" "}
+						{produce.shippingMaintenanceTemperatureLow} -{" "}
+						{produce.shippingMaintenanceTemperatureHigh}
+						<br />
+						Preset Temperature:{" "}
+						{produce.shippingPresetTemperature}
+						</Typography>
+					</Box>
+					<Box p={3}>
+						<Typography
+						className={classes.pos}
+						color="textSecondary"
+						>
+						Pricing (in USD / lb):
+						</Typography>
+						<Typography variant="body2" component="p">
+						USDA Price: ${produce.price}
+						<br />
+						Average Price Paid by Farmlink: $
+						{produce.pricePaid}
+						</Typography>
+					</Box>
+					<Box p={3}>
+						<Typography
+						className={classes.pos}
+						color="textSecondary"
+						>
+						Internal Statistics:
+						</Typography>
+						<Typography variant="body2" component="p">
+						Amount Moved by FarmLink (lbs):{" "}
+						{produce.amountMoved}
+						</Typography>
+					</Box>
+					</Box>
+				</CardContent>
+				<CardActions>
+					<Button
+					size="small"
+					color="primary"
+					onClick={() => this.handleViewOpen({ produce })}
+					>
+					{" "}
+					View{" "}
+					</Button>
+					<Button
+					size="small"
+					color="primary"
+					onClick={() =>
+						this.handleEditClickOpen({ produce })
+					}
+					>
+					Edit
+					</Button>
+					<Button
+					size="small"
+					color="primary"
+					onClick={() =>
+						this.deleteTodoHandler({ produce })
```

Comment:
might be time to rename this handler.

Reply:
Ah, sorry for the messiness - this is already renamed in master and it shows up accordingly in PR 10, in which i had pulled from master. I was trying to avoid re-resolving a bunch of merge conflicts that happened in PR 10 when I pulled from master, by merging PR 37's commits to PR 10. I just did this, and am in the process of dealing with these added merge conflicts but after I'm done, PR 10 should have the correct end version. I'm still not sure how to get my PR merged without dealing with the old files from the frontendConnected branch though...